Configurations/unix-Makefile.tmpl

```diff
@@ -741,56 +741,56 @@ lint:
 	lint -DLINT $(INCLUDES) $(SRCS)
 
 generate_apps:
-	( cd $(SRCDIR); $(PERL) VMS/VMSify-conf.pl \
+	( cd "$(SRCDIR)"; $(PERL) VMS/VMSify-conf.pl \
 				< apps/openssl.cnf > apps/openssl-vms.cnf )
 	@ : {- output_off() if $disabled{apps}; "" -}
-	( b=`pwd`; cd $(SRCDIR); \
+	( b=`pwd`; cd "$(SRCDIR)"; \
 	  $(PERL) -I$$b apps/progs.pl -H $(APPS_OPENSSL) > apps/progs.h )
-	( b=`pwd`; cd $(SRCDIR); \
+	( b=`pwd`; cd "$(SRCDIR)"; \
 	  $(PERL) -I$$b apps/progs.pl -C $(APPS_OPENSSL) > apps/progs.c )
```

Comment:
does -I$$b need quotes?

Reply:
You're right! Plus found another class of misses (some critical, some not).